### DISCUSSION TITLE
PackerComplete alike for lazy.nvim

### DISCUSSION THREAD

**Discussion by circleous** [NvChad/NvChad#1908] [Q&A]
Hi, I noticed that NvChad migrated to lazy.nvim recently. I use NvChad for my dev container, so Installs mostly are done unattended. I used PackerComplete event to trigger a quitall command, in Dockerfile it would be something like this

```Dockerfile
RUN nvim --headless -c 'autocmd User PackerComplete quitall'
``` 

So, after a quick search through lazy.nvim, I found this commit folke/lazy.nvim@a36d50639358bc00b8ac2d42a8a0a6c0f9c08310 
but I don't quite get what user event to use for full install and sync for first bootstrap. Any workaround or solution for that?

**Comment by siduck** [comment]
@circleous try LazyDone or LazySync
